_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -32,14 +32,18 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [WAI-ARIA state or property][] that is specified on an [HTML or SVG element][namespaced element] that is [included in the accessibility tree][].
+This rule applies to any [WAI-ARIA state or property][] that is:
+
+- **namespaced element**: specified on an [HTML or SVG element][namespaced element]; and
+- **perceivable**: the element is [included in the accessibility tree][]; and,
+- **not conflicting**: [WAI-ARIA state or property][] is not specified on an element that has the same [implicit WAI-ARIA semantic](https://www.w3.org/TR/wai-aria-1.2/#implicit_semantics).
```

Comment:
Following our [Listed conditions](https://act-rules.github.io/pages/design/rule-design/#listed-conditions) guidelines
```suggestion
This rule applies to any [WAI-ARIA state or property][] for which all the following is true:

- **namespaced element**: the property is specified on an [HTML or SVG element][namespaced element]; and
- **perceivable**: the property is specified on an element which is [included in the accessibility tree][]; and,
- **not conflicting**: the property is not specified on an element that has the same [implicit WAI-ARIA semantic](https://www.w3.org/TR/wai-aria-1.2/#implicit_semantics).
```
(also, you can use `<dfn>` element for the condition names and refer to them in the example descriptions)